data/templates/blb/config.yml

```diff
@@ -0,0 +1,8 @@
+name: Bad Luck Brian
+default:
+-
+link: http://knowyourmeme.com/memes/bad-luck-brian
+aliases:
+- Bad Luck Brian
```

Comment:
These should actually just be lowercase, hyphen-separated (like they'll be typed in a URL). Let me add this check to the validator to cause this PR to fail first.